Sources/CardPayments/CardClient.swift

```diff
@@ -43,6 +43,7 @@ public class CardClient: NSObject {
     /// If `didAttempt3DSecureVerification` is `true`, check verification status with `/v3/vault/setup-token/{id}` in your server.
     /// - Parameters:
     ///   - vaultRequest: The request containing setupTokenID and card
+    @available(*, deprecated, message: "This method is deprecated and will be removed in a future release. Use the new completion handler-based approach instead.")
```

Comment:
Maybe we can link them to the migration guide in our repo. Something like See our 2.0.0-beta2 migration guide in 
ttps://github.com/paypal/paypal-ios/

Reply:
Thanks, Victoria! That’s a very good point. I was also thinking about that. Let me implement it.

Reply:
Have you tried  adding options `run: pod lib lint --allow-warnings="deprecated"` in build.yml?